ui/qt-lib/src/Ariadne/UI/Qt/Widgets/Dialogs/Delete.hs

```diff
@@ -32,34 +37,76 @@ data Delete =
     , retypeWidget :: QWidget.QWidget
     , retypeName :: QLineEdit.QLineEdit
     , deleteButton :: QPushButton.QPushButton
+    , deleteUnknownObjects :: Maybe (QToolButton.QToolButton,QScrollArea.QScrollArea)
     , itemType :: UiDeletingItem
     }
 
+data MessagesOnDelWidget =
+  MessagesOnDelWidget
+    { header          :: !String
+    , intro           :: !String
+    , isSureMessage   :: !Text
+    , reTypeMessage   :: !String
+    , expandingMessge :: !Text
+    , confirmMsg      :: !Text
+    }
+
+makeMessages :: UiDeletingItem -> MessagesOnDelWidget
+makeMessages (UiDelUnknownKeys _) =
+  let header  = toString rmUnkownKeysHeaderMessage
+      intro = toString $ rmUnknownKeysIntroMessage
+      expandingMessge = "List of unknown secret keys"
+      isSureMessage = rmUnknownKeysSureMessage
+      reTypeMessage = toString $
+        rmUnknownKeysRetypeMkMessage ("<b>" <> rmUnknownKeysRetypeConfirm <> "</b>")
+      confirmMsg = rmUnknownKeysRetypeConfirm
+  in MessagesOnDelWidget{..}
+makeMessages (UiDelBrokenWallets _) =
+  let header  = toString rmBrokenWalletsHeaderMessage
+      intro = toString $ rmBrokenWalletsIntroMkMessage
+      expandingMessge = "List of wallets with a missed secret key"
```

Comment:
1. Why isn't this message (and `expandingMessge = "List of unknown secret keys"` above) part of `UiConfig`?
2. s/missed/missing (and there are many places in code where `missed` is used instead of `missing`).
3. I am not entirely sure, but probably it should be `with missing secret keys`, not `with a missing secret key`.